app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp, file1, file2 = command.split()
```

Comment:
Code Efficiency: You're assuming that the command will always be in the format 'cp file1 file2', but this might not always be the case. The command could have more than 3 parts, or less than 3 parts, or the 'cp' could be missing. You should add error checking for these cases.